bing_bert/ds_train_bert_bsz32k_seq512.sh

```diff
@@ -19,6 +19,7 @@ deepspeed ${base_dir}/deepspeed_train.py \
 --output_dir $OUTPUT_DIR \
 --print_steps 100 \
 --deepspeed \
+--deepspeed_transformer_kernel \
```

Comment:
just double confirm, we have decided to set kernel enabled by default?

Reply:
Yeah, I think it's probably fine to have our kernels enabled by default. This does set stochastic_model=False by default for pretraining it seems. I guess that's okay though right?

Reply:
I was thinking  of having the stable kernel as the default one, since it is less risky for the people to use it.

Reply:
I see, it is good.